package.json

```diff
@@ -36,6 +36,7 @@
     "@babel/preset-env": "7.5.5",
     "@babel/preset-react": "7.0.0",
     "@testing-library/react": "8.0.1",
+    "@testing-library/react-hooks": "^3.2.1",
```

Comment:
I used another testing-library for hooks so we don't need to write mock components to represent the results of hooks any more.